### PR TITLE
Implement a11yState and a11yValue queries

### PR DESCRIPTION
**Pull request by Esemesek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,25 +15,25 @@
     "integration"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
```
Comment (thymikee) [inline package.json]: can we bump dev deps in a separate PR to make it smaller?

Comment (Esemesek) [reply package.json]: Yup

**Comment by thymikee** [comment]
Can you make sure CI passes?

**Comment by Esemesek** [comment]
That is what I am working on right now :P

**Comment by Esemesek** [comment]
Will split this thing into couple of PRs

**Comment by thymikee** [comment]
Thanks, looking forward to it!

**Pull request closed without merging**